torq.sh

```diff
@@ -0,0 +1,263 @@
+    length=$(($#-$N-2));                                                                            # number of arguments minus extras 
+    array=${@:1:$length};                                                                           # arguments without extras 
+  else
+    array=$@;
+  fi
+ }
+
+getcsv() {
+  if [[ `echo ${BASH_ARGV[*]} | grep -e csv` ]]; then
+    eval flagcsv "csv";                                                                             # get csv file following flag
+    length=$(($#-2));                                                                               # number of arguments minus csv 
+    array=${@:1:$length};                                                                           # arguments without csv 
+    getprocs $array;
+  else
+    CSVPATH=${DEFAULTCSV};                                                                          # set csv file to default
+    getprocs $array;
+  fi
+ }
+
+getextrascsv() {
+  eval flagextras "extras";                                                                         # get arguments following flag
+  eval flagcsv "csv";
+  length=$(($#-$N-2));
+  array=${@:1:$length};                                                                             # arguments without extras and csv
+  getprocs $array;                                                                                  # get process names from the remaining arguments
+ }
+
+checkextrascsv() {
+  if [[ `echo ${BASH_ARGV[*]} | grep -e extras` ]] && [[ `echo ${BASH_ARGV[*]} | grep -e csv` ]]; then
+    getextrascsv $@;                                                                                # gets extras and csv arguments 
+  else
+    getextras $@;                                                                                   # checks if extras flag present
+    getcsv $@;                                                                                      # sets process csv file
+  fi
+ }
+
+allcsv() {
+  if [[ `echo ${BASH_ARGV[*]} | grep -e csv` ]]; then
+    eval flagcsv "csv";                                                                             # get all procs from csv without starting 
+  else
+    CSVPATH=${DEFAULTCSV};                                                                            
+  fi
+ }
+
+startprocs() {
+  for p in $PROCS;
+  do
+    start $p;                                                                                       # start each process in variable
+  done
+ }
+
+stopprocs() {
+  for p in $PROCS;
+  do
+    stop $p;                                                                                        # kill each process in variable 
+  done
+ }
+
+usage() {
+  printf -- "Arguments:\n"
+  printf -- "  start all                                to start all processes\n"
+  printf -- "  start <processname(s)>                   to start process(es)\n"
```

Comment:
condense 212 and 213 into something like `start all|<processname(s)>`, this seems kinda redundant as-is. Same with the `stop` commands on 215 & 216